NEWS.md

```diff
@@ -64,6 +64,16 @@ reduce2_right(.x = letters[1:4], .y = paste2, .f = c("-", ".", "-")) # working
 
 ## Minor improvements and fixes
 
+* Functions taking predicates (`map_if()`, `keep()`, `some()`,
```

Comment:
I think `TRUE` or `FALSE` is probably more accurate and easier to grasp than "scalar logical"